languages/elixir/exercises/concept/enum/.docs/introduction.md

```diff
@@ -0,0 +1,60 @@
+`Enum` is a very useful module that provides a set of algorithms for working with enumerables. It offers sorting, filtering, grouping, counting, searching, finding min/max values, and much more.
+
+In Elixir, an enumerable is any data type that implements the `Enumerable` protocol. The most common of those are lists and maps.
```

Comment:
Would it be useful to explain what an enumerable is? Something along the lines of an enumerable being something that can be iterated over, like a collection. Or do students already know this?